ethpm_types/contract_type.py

```diff
@@ -204,6 +205,44 @@ def extract_sourcemap_item(expanded_row, item_idx, previous_val=None):
             yield item
 
 
+class PCMapItem(BaseModel):
+    line_start: Optional[int]
+    column_start: Optional[int]
+    line_end: Optional[int]
+    column_end: Optional[int]
+
+
+class PCMap(BaseModel):
+    """
+    As part of the source output, the compiler provides a map of program counter values to
```

Comment:
Should we mention that vyper is example compiler making use of this type?

Reply:
@unparalleled-js It's in Vyper and not in Solidity right now, but we're going to add pc-map construction ourselves to `ape-solidity` (and maybe `ape-cairo`) later

So while it's Vyper-only right now, here in ethpm-types the `PCMap` and `PCMapItem` stuff will be general and not compiler specific

Reply:
Ah okay, understood.
I would not worry about `ape-cairo` too much for the time being, besides capturing it in a ticket there.

Reply:
Yup! Got in APE-298 for Solidity and APE-299 for Cairo 🙂